packages/strapi-helper-plugin/lib/internals/webpack/webpack.dev.babel.js

```diff
@@ -16,19 +16,11 @@ const LodashModuleReplacementPlugin = require('lodash-webpack-plugin');
 
 const isAdmin = process.env.IS_ADMIN === 'true';
 const isSetup = path.resolve(process.env.PWD, '..', '..') === path.resolve(process.env.INIT_CWD);
```

Comment:
Maybe this should be `const isSetup = process.env.IS_MONOREPO || false;`, same as in other files 🤔